src/oneLine.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default function OneLine({ element, index, onClickComplete }) {
+  return (
+    <li>
+      {element}
+      <button type="button" onClick={() => onClickComplete(index)}>완료</button>
```

Comment:
```suggestion
      <button
         type="button"
         onClick={() => onClickComplete(index)}
     >
         완료
     </button>
```
차이점을 비교해보면 어떤게 더 나으신가요?

저는 `props` 가 많을 때에는 줄바꿈을 해서 가독성을 높이려고 하는 편이에요.

보시고 개인적으로 어떤게 더 나은지 의견 나눠보면 좋을 것 같아요!

Reply:
넵! 저번과제에서도 지도해주셨는데 또 틀렸네요 ㅠㅠ 바로 바꾸겠습니다!